packaging/convert2rhel.spec

```diff
@@ -127,7 +127,7 @@ install -p man/%{name}.8 %{buildroot}%{_mandir}/man8/
   the packager when the vendor information is not available.
 
 * Fri Jan 8 2021 Michal Bocek <mbocek@redhat.com> 0.15-1
-- add missing CentOS 8 packages to be removed prior the conversion
+- add missing CentOS Linux 8 packages to be removed prior the conversion
```

Comment:
Please do not change the changelog entries. Let's consider it as immutable - not to be changed in the future.

Reply:
@hosekadam, please revert also the changes in the other changelog entries. Thanks.